Readme.md

```diff
@@ -29,19 +29,19 @@ render backends it only focuses on the actual UI.
 
 ## Building
 
-This library is self contained in one single header file and can be used either
-in header only mode or in implementation mode. The header only mode is used
+This library is self-contained in one single header file and can be used either
+in header-only mode or in implementation mode. The header-only mode is used
 by default when included and allows including this header in other headers
 and does not contain the actual implementation.
 
-The implementation mode requires to define  the preprocessor macro
+The implementation mode requires defining the preprocessor macro
 `NK_IMPLEMENTATION` in *one* .c/.cpp file before `#include`ing this file, e.g.:
 ```c
 #define NK_IMPLEMENTATION
 #include "nuklear.h"
 ```
 IMPORTANT: Every time you include "nuklear.h" you have to define the same optional flags.
-This is very important not doing it either leads to compiler errors or even worse stack corruptions.
+This is very important not doing it either leads to compiler errors, or even worse, stack corruptions.
```

Comment:
@sglass68: I think this also needs a semi-colon or colon to separate "This is very important" from the rest of the sentence.

Reply:
yes